src/audio/kpb.c

```diff
@@ -56,24 +56,24 @@ DECLARE_SOF_UUID("kpb-task", kpb_task_uuid, 0xe50057a5, 0x8b27, 0x4db4,
 
 /* KPB private data, runtime data */
 struct comp_data {
-	uint64_t state_log; /**< keeps record of KPB recent states */
 	enum kpb_state state; /**< current state of KPB component */
-	uint32_t kpb_no_of_clients; /**< number of registered clients */
-	struct kpb_client clients[KPB_MAX_NO_OF_CLIENTS];
-	struct task draining_task;
-	uint32_t source_period_bytes; /**< source number of period bytes */
-	uint32_t sink_period_bytes; /**< sink number of period bytes */
+	uint64_t state_log; /**< keeps record of KPB recent states */
```

Comment:
Can you explain why in the commit message. I assume it's putting items closer together that are used together for faster access / better cache perf ?

Reply:
Yes, that's it. The commit message has been updated.